flink-runtime/src/main/java/org/apache/flink/runtime/executiongraph/ExecutionJobVertex.java

```diff
@@ -227,6 +229,10 @@ public ExecutionState getAggregateState() {
 		
 		return getAggregateJobVertexState(num, parallelism);
 	}
+
+	public boolean receivedStopSignal() {
```

Comment:
The old WebUI used it (this code got delete when we switched to the new UI) -- I guess it is obsolete now. Will remove it.